go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/prometheus/exporter-toolkit
 
-go 1.20
+go 1.21
 
 require (
 	github.com/alecthomas/kingpin/v2 v2.4.0
 	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/go-kit/log v0.2.1
-	github.com/prometheus/common v0.46.0
-	golang.org/x/crypto v0.20.0
+	github.com/prometheus/common v0.50.0
```

Comment:
I'm not too familiar with all that exporter toolkit does, does it add any metrics to your application? If yes, we probably want to avoid common 0.50 🤔

Reply:
Oh, oops, I made this weeks ago. :joy_cat: